src/Microsoft.Azure.SignalR.Common/IMessageWithTracingIdExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Threading;
+using Microsoft.Azure.SignalR.Protocol;
+
+namespace Microsoft.Azure.SignalR
+{
+    internal static class IMessageWithTracingIdExtensions
+    {
+        private static int _prefix = Guid.NewGuid().GetHashCode();
+        private static int counter = 0;
+
+        public static void UpdateTracingId(this IMessageWithTracingId msg)
+        {
+            msg.TracingId = $"{(uint)_prefix}:{(uint)counter}";
```

Comment:
prefix to reuse serverName? maybe a Id generator

Also I think in netcore it choose to use the naming convention `Logging` instead `Tracing`

TracingId is string, shall we use a fixed length for counter?

Reply:
Better to use something unrelated to customer's custom name, to avoid privacy issue in service.
Fixed length sounds good, also make it a HEX to shortern the length.

Reply:
Reuse this one https://github.com/Azure/azure-signalr/blob/dev/src/Microsoft.Azure.SignalR.Common/DefaultServerNameProvider.cs#L13

> to avoid privacy issue in service
Hm, if privacy is a concern, we need to update the default server name provider's logic instead. cc @JialinXin

Reply:
Right, Guid itself is good, do not add server name. We'd update the provider's logic.

Reply:
updated

Reply:
updated